src/main/java/com/wanderersoftherift/wotr/mixin/MixinItem.java

```diff
@@ -15,6 +18,14 @@ public abstract class MixinItem {
     @Inject(method = "onCraftedPostProcess", at = @At("TAIL"))
     public void addCustomDataOnCraftedPostProcess(ItemStack stack, Level level, CallbackInfo ci) {
         // Add custom data to the ItemStack when it is crafted
-        GearSockets.generateForItem(stack, level, 1, 3);
+        if (level.isClientSide()) {
```

Comment:
Might try using ItemCraftedEvent rather than a mixin

Reply:
@CSTufts did you consider this when you made your change?

Reply:
I do remember that ItemCraftedEvent currently does not fire on the Auto Crafter.

Reply:
Mmm... looks like this is in the best place already.